src/components/MaAlert/MaAlert.vue

```diff
@@ -1,39 +1,58 @@
 <template>
   <div :class="['alert-banner', getClasses]">
     <span class="alert-banner__icon" />
-    <div class="alert-banner__content">
-      <p v-if="title" class="alert-banner__title" v-text="title" />
+    <ma-stack space="xsmall">
+      <ma-heading v-if="title" size="xsmall" v-text="title" />
+      <!-- @slot Alert content slot -->
       <slot>
-        <p class="alert-banner__text" v-text="text" />
+        <ma-text v-if="text" tag="p" v-text="text" />
       </slot>
-    </div>
+    </ma-stack>
   </div>
 </template>
 
 <script>
+import MaStack from '@margarita/components/MaStack'
+import MaHeading from '@margarita/components/MaHeading'
+import MaText from '@margarita/components/MaText'
 const AVAILABLE_SIZES = ['small', 'medium', 'large']
 const AVAILABLE_TYPES = ['error', 'info', 'success', 'warning']
 
 export default {
   name: 'MaAlert',
 
+  components: {
+    MaStack,
+    MaHeading,
+    MaText,
+  },
+
   props: {
+    /**
+     * Sets the size of the component
+     */
     size: {
       default: 'medium',
       type: String,
       validator: (value) => AVAILABLE_SIZES.includes(value),
```

Comment:
For now, vue-docgen-api (and thus Vetur) are unable to get the right values from a variable, so they can't provide them as autocomplete options.

However, this works:

```suggestion
      validator: (value) => ['small', 'medium', 'large'].includes(value),
```


It is a known limitation and [we're working on it](https://github.com/vue-styleguidist/vue-styleguidist/pull/1051) :)

Reply:
I do believe we could do the same one the `type` property.
```js
// from:
validator: (value) => AVAILABLE_TYPES.includes(value),

// to:
validator: (value) => ['error', 'info', 'success', 'warning'].includes(value),
```